src/libs/events.ts

```diff
@@ -124,6 +124,12 @@ const eventsList = {
   workflowsTabView: 'workflowsApp:tab:view',
   workspaceClone: 'workspace:clone',
   workspaceCreate: 'workspace:create',
+  workspaceDashboardSectionToggle: 'workspace:dashboard:toggleSection',
```

Comment:
Nit: DIsconnect between the key name and the event name: is it `SectionToggle` or `toggleSection`?

Reply:
Will fix.